KE_test/models.py

```diff
@@ -0,0 +1,76 @@
+from django.db import models
+
+
+# Create your models here.
+
+# TODO:сделать id лонгами?
+
+
+class Shop(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+    image = models.ImageField(upload_to="shop_images/", null=True, blank=True)
+
+    class Meta:
+        verbose_name = "Shop"
+        verbose_name_plural = "Shops"
+
+    def __str__(self):
+        return self.title
+
+
+class Product(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True, blank=True)
+    amount = models.IntegerField()
+    price = models.FloatField()
+
+    # the minimal requirements say that amount, price and images should be in
+    # "Product" table, so I guess I'm not able to change that.
+    # However, if I could, I would make a separate "ProductShop", which would have
+    # shop_id, product_id, amount and price.
+    # It would lead to less data duplication (because product name, description and images
```

Comment:
Maybe I didn't get the point, but how would the initial conditions lead to data duplication?
In this case product has its own and separate from shop title, description and set of images. Imagine that we have a shop called `Apple`. It may have its own image as a title image, and some description. Inside it has a set of products, one of which is called `iPhone 7`. This product card has its own title, description and images, which are not related to Shop entity.

Reply:
If it was not understandable from the task, sorry 😊

Reply:
What I meant is:
As I understood from the task, we create a whole separate product for each shop => if "Apple California shop" and "Apple NY shop" have exactly the same product "iPhone 7 RED" in them, it will be duplicated. I attached an image of the schema that would reduce that duplication (we will have only a subset of attributes duplicated for each shop, but images, for example, will be recorded only once).

![image](https://user-images.githubusercontent.com/35569740/65258631-dbc8bd80-db0b-11e9-9114-f7f89fec3d17.png)

Reply:
Here is more visual description of what I mean. Also, `orders` and `active` can be moved to `Shop_Product` table, but the task does not give enough details to make that decision 
(can the product be `active=true' in one shop and `active=false` in another shop? Do we count `orders` globally or per-shop?)

![image](https://user-images.githubusercontent.com/35569740/65259420-4fb79580-db0d-11e9-8c7e-d3d69264cf02.png)

Reply:
Now I see your point.
For the conditions that you have provided, your solution is of course better. Thanks for clarification. 
I assumed that only one instance of a brand product exists (because this is how it is done in our field of development)